changehc/delphi_changehc/update_sensor.py

```diff
@@ -62,10 +62,8 @@ def write_to_csv(df, geo_level, write_se, day_shift, out_name, logger, output_pa
         sensor=out_name,
         write_empty_days=True
     )
-    logger.debug("wrote {0} rows for {1} {2}".format(
-        df.size, df["geo_id"].unique().size, geo_level
-    ))
-    logger.debug("wrote files to {0}".format(output_path))
+    logger.debug("Wrote rows", num_rows=df.size, geo_type=geo_level)
```

Comment:
might as well keep the other value we logged before:
```suggestion
    logger.debug("Wrote rows", num_rows=df.size, geo_type=geo_level, num_geo_ids=df["geo_id"].unique().size)
```